rpc/cc/arduino/cli/commands/v1/compile.proto

```diff
@@ -97,26 +97,33 @@ message CompileRequest {
 }
 
 message CompileResponse {
-  // The output of the compilation process (stream)
-  bytes out_stream = 1;
-  // The error output of the compilation process (stream)
-  bytes err_stream = 2;
+  oneof message {
+    // The output of the compilation process (stream)
+    bytes out_stream = 1;
+    // The error output of the compilation process (stream)
+    bytes err_stream = 2;
+    // Completions reports of the compilation process (stream)
+    TaskProgress progress = 3;
+    // The compilation result
+    BuilderResult result = 4;
+  }
+}
+
+message BuilderResult {
```

Comment:
Please consider `CompileResult`. It fits better with the `Compile` service method naming and `CompileRequest`/`CompileResponse` names.

Reply:
I opted for `BuilderResult` to keep consistency with the `arduino-cli compile` JSON output that already has a field named `builder_result`.